.pre-commit-config.yaml

```diff
@@ -33,7 +33,6 @@ repos:
     rev: 1.13.0
     hooks:
     -   id: blacken-docs
-        additional_dependencies: [black==21.11b1]
```

Comment:
can't we just set it to 22.12.0 (like is used above)? this *is* an additional dependency that's required for this hook, i don't think we should remove it outright

Reply:
i see VERY little point in adding the dependency there when black itself is already in the same file as a dependency. this is just a liability that we have to maintain on every update at this point

Reply:
fair

Reply:
i would be fine with adding it back if pre-commit ci would handle it but it doesn't sadly